src/asmcnc/apps/systemTools_app/screens/popup_system.py

```diff
@@ -601,6 +601,45 @@ def reboot_console(*args):
 
         popup.open()
 
+class PopupSSHToggleFailed(Widget):
+    def __init__(self):
+        description = 'Please contact Yeti Tool support'
```

Comment:
This is a bit too direct - if someone accidentally presses this button and ends up seeing this popup, they WILL contact us anyway! 😅 

- What are the reasons why enabling/disabling ssh might fail? 
- If it does fail, might be worth advising users to safely reboot their console first in case this fixes things
- Is there anything else they can do?

Reply:
Not sure why it could fail - I haven't been able to cause it to happen.

I guess they can try restarting.